Names-values.Rmd

```diff
@@ -694,43 +694,43 @@ rm(x)
+R uses a __tracing__ GC. This means it traces every object that's reachable from the global[^callstack] environment, and all objects that are, in turn, reachable from those objects (i.e. the references in lists and environments are searched recursively). The garbage collector does not use the modify-in-place reference count described above. While these two ideas are closely related, the internal data structures are optimised for different use cases.
 
-[^callstack]: And every environment on the current call stack.
+[^callstack]: And every environment in the current call stack.
 
-The garbage collector (GC) is run automatically whenever R needs more memory to create a new object. From the outside, it's basically impossible to predict when the GC will run, and indeed, you shouldn't try. Instead, if you want to find out when the GC runs, call `gcinfo(TRUE)`: then the GC will print a message to the console every time it runs.
+The garbage collector (GC) runs automatically whenever R needs more memory to create a new object. Looking from the outside, it's basically impossible to predict when the GC will run. In fact, you shouldn't even try. If you want to find out when the GC runs, call `gcinfo(TRUE)` and GC will print a message to the console every time it runs.
 
-You can force the garbage collector to run by calling `gc()`. Despite what you might have read elsewhere, there's never any _need_ to call `gc()` yourself. You may _want_ to call `gc()` to ask R to return memory to your operating system, or for its side-effect of telling you how much memory is currently being used:  
+You can force garbage collection by calling `gc()`. But despite what you might have read elsewhere, there's never any _need_ to call `gc()` yourself. The only reasons you might _want_ to call `gc()` is to ask R to return memory to your operating system, or is for the side-effect that tells you how much memory is currently being used:   
 
 <!-- GVW: why would I want to ask R to return memory the OS? -->
 
 ```{r}
 gc() 
 ```
 
-`lobstr::mem_used()` is a wrapper around `gc()` that just prints the total number of bytes used:
+`lobstr::mem_used()` is a wrapper around `gc()` that prints the total number of bytes used:
 
 ```{r}
 mem_used()
 ```
 
-This number won't agree with the amount of memory reported by your operating system for three reasons:
+This number won't agree with the amount of memory reported by your operating system. There are three reasons:
 
-1. It only includes objects created by R, not the R interpreter itself.
+1. It only includes objects created by R, not the R interpreter.
```

Comment:
This makes it sound like it doesn't include objects created by the R interpreter.